File: Resource/51900286-51900712-51900718-51900699/source demo/Web Application/ComNha_API/ComNha_API/Models/Hoivien.cs
using System;
using System.Collections.Generic;

namespace ComNha_API.Models
{
    public partial class Hoivien
    {
        public Hoivien()
        {
            Hoadons = new HashSet<Hoadon>();
        }

        public string TenHoiVien { get; set; } = null!;
        public string SoDienThoai { get; set; } = null!;
        public double TongSoTienThanhToan { get; set; }
        public string Email { get; set; } = null!;
        public string MatKhau { get; set; } = null!;
        public string DiaChi { get; set; } = null!;
        public string QuyenLoi { get; set; } = null!;

        public virtual ICollection<Hoadon> Hoadons { get; set; }
    }
}

File: References/restaurent-management-comnha/src/main/java/tdtu/thinh6/restaurent_management_comnha/domain/Hoivien.java
package tdtu.thinh6.restaurent_management_comnha.domain;

import java.util.Set;
import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.Id;
import javax.persistence.OneToMany;
import lombok.Getter;
import lombok.Setter;


@Entity
@Getter
@Setter
public class Hoivien {

    @Id
    @Column(nullable = false, updatable = false, length = 20)
    private String soDienThoai;

    @Column(nullable = false, length = 50)
    private String tenHoiVien;

    @Column(nullable = false)
    private Double tongSoTienThanhToan;

    @Column(nullable = false, length = 100)
    private String email;

    @Column(nullable = false, length = 100)
    private String diaChi;

    @Column(nullable = false, length = 100)
    private String quyenLoi;

    @OneToMany(mappedBy = "maHoiVien")
    private Set<Hoadon> maHoiVienHoadons;

}
